### ISSUE TITLE
Issues with Python to make clear to users

### ISSUE THREAD

**Issue by tallguyjenks** [Pseudonium/Obsidian_to_Anki#14]
with #13 that might help if some have a less stupid implementation of python on their system, i used brew on mac to install it and so the shebang causes issues with that, but it would still be a great addition to portably let people just execute it. For FAQ or installation instructions you could also say that if you hit errors like i did, to just explicitly run `python <path to script>` compared to current readme: 
>From the command line, run the script once with no arguments - {Path to script}/obsidian_to_anki.py This will make a configuration file in the same directory as the script, "obsidian_to_anki_config.ini".

just a thought ^_^

**Comment by Pseudonium** [comment]
Yep, added a troubleshooting section that suggests this as a fix. Thanks!

**Comment by tallguyjenks** [comment]
welcome! since i got this working, i chose to keep anki and just purchase the mobile app, and im glad i went with that. thanks for this great tool!

**Comment by Pseudonium** [comment]
Glad to hear that you find it useful! And I hope Anki serves you well.
